src/main.ts

```diff
@@ -7,15 +7,20 @@ import * as core from '@actions/core'
 import * as github from '@actions/github'
 import {GitHub} from '@actions/github/lib/utils'
 
+const GITHUB_API_URL = 'https://api.github.com'
+
 function createOctokit(): Octokit {
   const token = core.getInput('token')
+  const gitHubOptions = {
+    baseUrl: core.getInput('github-api-url') || GITHUB_API_URL
```

Comment:
Can we specify a default in the action.yml file instead? It avoids the need for the extra const and conditional.